elasticsearch/serializer.py

```diff
@@ -37,7 +50,10 @@ def default(self, data):
 
     def loads(self, s):
         try:
-            return json.loads(s)
+            obj = json.loads(s)
+            if self._is_repl:
+                return _PrettyJSONDict(obj)
```

Comment:
This will fail for APIs that return a `list`, for example `es.cat.nodes(format='json')`

Reply:
Oh this wasn't meant to be in this PR, only meant for my own usage working through the "Getting Started" documentation. :) Will revert these changes.